app/Controllers/importExportController.php

```diff
@@ -64,31 +64,24 @@ public function importFile($name, $path, $username = null) {
 
 		// We try to list all files according to their type
 		$list = array();
-		if ($type_file === 'zip' && extension_loaded('zip')) {
-			$zip = zip_open($path);
-			if (!is_resource($zip)) {
+		if ('zip' === $type_file && extension_loaded('zip')) {
+			$zip = new ZipArchive();
+			$result = $zip->open($path);
```

Comment:
```suggestion
			$result = $zip->open($path, 16);	//ZipArchive::RDONLY from PHP 7.4.3+
```

What about this suggestion?

Reply:
I feel that it's not needed. I can add it if you prefer though